CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Enhancements
 
+* Added support for using non-encrypted Realm in multiple processes from the same apk (#1091).
```

Comment:
This is slightly pessimistic as using `sharedUserId` allows multiple APK's to access it, but I'm not entirely sure how to reword it to make it clear since there is also a bunch of other caveats. Perhaps something likes

```
Added support for using non-encrypted Realms in multiple processes. Some caveats apply. Read <insertLinkToDocs> for more info (#1091).
```